bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeConverter.java

```diff
@@ -267,6 +270,15 @@ static boolean isConvertibleToFloatingPointTypes(Object value) {
         return convertibleTypes;
     }
 
+    private static boolean checkIsType(Object value, Type targetType) {
```

Comment:
Shall we call this `checkIsSameType` instead? Because `checkIsType` means something else in `TypeChecker`?

Reply:
+1. Will do

Reply:
Should we just move this in-line?

```
TypeChecker.getType(value) == targetType
```